lib/rspec/core/formatters/bisect_formatter.rb

```diff
@@ -0,0 +1,68 @@
+require 'drb/drb'
+
+module RSpec
+  module Core
+    module Formatters
+      # Used by `--bisect`. When it shells out and runs a portion of the suite, it uses
+      # this formatter as a means to have the status reported back to it, via DRb.
+      #
+      # Note that since DRb calls carry considerable overhead compared to normal
+      # method calls, we try to minimize the number of DRb calls for perf reasons,
+      # opting to communicate only at the start and the end of the run, rather than
+      # after each example.
+      # @private
+      class BisectFormatter
+        Formatters.register self, :start, :start_dump, :example_started,
+                            :example_failed, :example_passed, :example_pending
+
+        def initialize(_output)
+          port                = RSpec.configuration.drb_port
+          drb_uri             = "druby://127.0.0.1:#{port}"
```

Comment:
`127.0.0.1` refers to localhost on every platform, right?

Bisect is designed to communicate over DRb to a localhost server so it's specifically hardcoded to the localhost ip.